code/modules/mob/living/silicon/decoy/decoy.dm

```diff
@@ -30,6 +30,8 @@
 	icon_state = "ai-magma"
 
 /mob/living/silicon/decoy/syndicate/depot
+	universal_speak = 1

```

Comment:
Shouldn't this be TRUE, same for understand?

Reply:
Changed